doc/modules/multi_column_operations/column_selection_pipeline.rst

```diff
@@ -0,0 +1,10 @@
+Column selection inside a pipeline
+----------------------------------
```

Comment:
```suggestion
----------------------------------

.. currentmodule:: skrub

```

This is important so that the symbols actually turn into links to the API documentation (very important for the user)